website/server/controllers/api-v3/user.js

```diff
@@ -1310,4 +1310,28 @@ api.userReset = {
   },
 };
 
+/**
+* @api {post} /api/v3/user/set-custom-day-start Sets preferences.dayStart for user
+* @apiVersion 3.0.0
+* @apiName UserSetCustomDayStart
+* @apiGroup User
+*
+* @apiSuccess {Object} data An empty Object
+*/
+api.userSetCustomDayStart = {
+  method: 'POST',
+  middlewares: [authWithHeaders()],
+  url: '/user/set-custom-day-start',
+  async handler (req, res) {
+    let user = res.locals.user;
+    let dayStart = req.body.dayStart;
```

Comment:
There is validation for this on the user model but I'd add an is empty check anyway because it's not marked as required